xDripTests/Scenes/Settings/AlertSettings/SettingsAlertSingleType/SettingsAlertSingleTypeViewControllerTests.swift

```diff
@@ -87,39 +87,41 @@ final class SettingsAlertSingleTypeViewControllerTests: XCTestCase {
     }
     
     func testTableView() {
+        sut.router?.dataStore?.eventType = .fastRise
         loadView()
         
         let configuration = sut.router?.dataStore?.configuration
         
         XCTAssertTrue(tableView.numberOfSections == 2)
-        XCTAssertTrue(tableView.numberOfRows(inSection: 0) == 1)
+        XCTAssertTrue(tableView.numberOfRows(inSection: 0) == 2)
         
         // When
         toggleSecondSection(expanded: false)
         // Then
         XCTAssertTrue(configuration?.isOverriden == false)
-        XCTAssertTrue(tableView.numberOfRows(inSection: 1) == 1)
+        XCTAssertTrue(tableView.numberOfRows(inSection: 1) == 5)
         
         // When
         toggleSecondSection(expanded: true)
         // Then
         XCTAssertTrue(configuration?.isOverriden == true)
-        XCTAssertTrue(tableView.numberOfRows(inSection: 1) == 7)
+        XCTAssertTrue(tableView.numberOfRows(inSection: 1) == 11)
         
         // When
         toggleEntireDaySwitch(expanded: true, isSecondSectionExpanded: true)
         // Then
         XCTAssertTrue(configuration?.isEntireDay == true)
-        XCTAssertTrue(tableView.numberOfRows(inSection: 1) == 7)
-        
+        XCTAssertTrue(tableView.numberOfRows(inSection: 1) == 11)
+
```

Comment:
Why do we need toggle Entire Day Switch if nothing happens?

Reply:
It is enabled by default, maybe should remove this test case.